joomla/joomla-cms
# Fixed "answer provided by user" issue on reCAPTCHA

Pull Request for Issue # .
"answer provided by user" issue on reCAPTCHA, Specially it's conflicting if multiple reCAPTCHA found on the respective page.

### Summary of Changes
Worked on "onCheckAnswer" function. The "$code" parameter's value wasn't implementing inside the function. While you call "onCheckAnswer" function, it used to take "response" from the form which was conflicting if multiple reCAPTCHA found on the respective page.

### Testing Instructions
Create multiple contact forms in one page with reCAPTCHA (it should be ajax contact form, We tested by SP Page Builder) and submit the second or third form with complete reCAPTCHA.

### Expected result
if anyone sends any value to "$code" parameter on "onCheckAnswer" function, it should take the "$code" parameter's value "$code" as "response".

### Actual result
Worked on "onCheckAnswer" function. The "$code" parameter's value wasn't implementing inside the function. While you call "onCheckAnswer" function, it used to take "response" from the form which was conflicting if multiple reCAPTCHA found on the respective page.

### Documentation Changes Required
No
